aptos-move/block-executor/src/txn_commit_hook.rs

```diff
@@ -19,21 +18,21 @@ pub struct NoOpTransactionCommitHook<T, E> {
     phantom: std::marker::PhantomData<(T, E)>,
 }
 
-impl<T: TransactionOutput, E: Debug + Sync + Send> Default for NoOpTransactionCommitHook<T, E> {
+impl<T: TransactionOutput, E: Sync + Send> Default for NoOpTransactionCommitHook<T, E> {
     fn default() -> Self {
         Self::new()
     }
 }
 
-impl<T: TransactionOutput, E: Debug + Sync + Send> NoOpTransactionCommitHook<T, E> {
+impl<T: TransactionOutput, E: Sync + Send> NoOpTransactionCommitHook<T, E> {
```

Comment:
Why remove debug?

Reply:
Having lesser trait bounds means it is more generic.